tests/unittests/tests-gnrc_ipv6/tests-gnrc_ipv6.c

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (C) 2016 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @author      Oliver Hahm <oliver.hahm@inria.fr>
+ * @file
+ */
+
+#include "embUnit.h"
+
+#include "net/ipv6/addr.h"
+#include "net/ipv6/hdr.h"
+#include "net/gnrc/ipv6.h"
+#include "net/gnrc/pktbuf.h"
+
+#include "unittests-constants.h"
+#include "tests-gnrc_ipv6.h"
+
+#define DEFAULT_TEST_SRC    { { \
+            0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, \
+            0x08, 0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f \
+        } \
+    }
+#define DEFAULT_TEST_DST    { { \
+            0x10, 0x11, 0x12, 0x13, 0x14, 0x15, 0x16, 0x17, \
+            0x18, 0x19, 0x1a, 0x1b, 0x1c, 0x1d, 0x1e, 0x1f \
+        } \
+    }
+
+#define DEFAULT_TEST_V_TC_FL    TEST_UINT32
+#define DEFAULT_TEST_LEN        TEST_UINT16
+#define DEFAULT_TEST_NH         TEST_UINT8
+#define DEFAULT_TEST_HL         TEST_UINT8
+
+static gnrc_pktsnip_t *_pkt_w_ip_hdr, *_pkt_no_ip_hdr;
+
+static void set_up(void)
+{
+    ipv6_hdr_t ip = (ipv6_hdr_t) {
+        .v_tc_fl    = byteorder_htonl(DEFAULT_TEST_V_TC_FL),
+        .len        = byteorder_htons(DEFAULT_TEST_LEN),
+        .nh         = DEFAULT_TEST_NH,
+        .hl         = DEFAULT_TEST_HL,
+        .src        = DEFAULT_TEST_SRC,
+        .dst        = DEFAULT_TEST_DST
+    };
+
+    ipv6_hdr_set_version(&ip);
+
+    gnrc_pktbuf_init();
+    _pkt_w_ip_hdr = gnrc_pktbuf_add(NULL, NULL, 1, GNRC_NETTYPE_NETIF);
+    assert(_pkt_w_ip_hdr);
```

Comment:
Because it is the setup function and this would conflict with my understanding of how a unit test is supposed to work (one test == one test/check/assertion).